src/Symfony/Component/Stopwatch/Section.php

```diff
@@ -80,9 +78,9 @@ public function get($id)
      *
      * @return self
      */
-    public function open($id)
+    public function open(?string $id)
```

Comment:
maybe like this so `open()` can create new session and `open(id)` re-opens with given id?
and define behavior if id is invalid (i'd rather throw logic exception)

```suggestion
    public function open(string $id = null)
```

Reply:
While it might make sense, it is not related to this PR and would need to be done in 4.4